server/graphql/loaders/comment-reaction.ts

```diff
@@ -0,0 +1,17 @@
+import DataLoader from 'dataloader';
+import models from '../../models';
+import { Op } from 'sequelize';
+
+/**
+ * Loader for all collective's payout methods
+ */
+export default {
+  generateCommentReactionsLoader: (): DataLoader<number, object> =>
```

Comment:
We could add this to `server/graphql/loaders/comments.ts` as `reactionsByCommentId`